app/support/products/scheduling_support.rb

```diff
@@ -98,6 +98,10 @@ def offline?
     offline_reservations.current.any?
   end
 
+  def offline_category
```

Comment:
This fields a little weird in `SchedulingSupport`, but the whole separation into concerns is already a little weird.